webpack.config.js

```diff
@@ -25,6 +25,9 @@ const baseConfig = {
         loader: 'json-loader',
 
       },
+      {
+        test: /\.(png|jpe?g|eot|ttf|woff|woff2|svg)$/, exclude: [/dist\//, /node_modules/], loader: 'file-loader', query: { name: '[name]-[hash:base64:5].[ext]' },
```

Comment:
Webfonts afaik do not need minification as is, as long as they are served gzipped.